src/imgtools/ops/functional.py

```
@@ -1,5 +1,5 @@
 from collections import namedtuple
-from typing import Optional, Sequence, Tuple, Union
+from typing import Optional, Sequence, Tuple, Union, List
```

Ruff (0.8.2)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)